src/app/search-bar/search-bar.component.html

```diff
@@ -1,75 +1,67 @@
-          <span
-            *ngIf="creator.IsReserved && !creator.IsVerified"
-            class="d-inline-block ml-1 cursor-pointer lh-12px fc-muted"
-            matTooltipClass="global__mat-tooltip global__mat-tooltip-font-size"
-            [matTooltip]="'This profile is reserved. Click to learn how to claim it.'"
-          >
-            <i class="far fa-clock fa-md align-middle"></i>
-          </span>
-          <span
-            *ngIf="creator.IsVerified"
-            (click)="tooltip.toggle()"
-            class="ml-1 cursor-pointer text-primary"
-            matTooltipClass="global__mat-tooltip global__mat-tooltip-font-size"
-            [matTooltip]="'This account is verified'"
-            #tooltip="matTooltip"
-          >
-            <i class="fas fa-check-circle fa-md align-middle"></i>
-          </span>
-          <span *ngIf="creator.CoinPriceBitCloutNanos">&nbsp;&middot;&nbsp;</span>
+    <div class="fs-15px">
+      <div *ngIf="!loading && creators.length > 0">
+        <div *ngFor="let creator of creators; let ii = index" class="d-flex align-items-center"
+          [ngClass]="{ 'search-bar__selected-color': creator.Username == creatorSelected }"
+          (click)="_handleCreatorSelect(creators[selectedCreatorIndex])"
+          (mouseover)="_handleMouseOver(creator.Username, ii)">
+          <div class="search-bar__avatar m-10px" [avatar]="creator.PublicKeyBase58Check"></div>
+          <div>
+            <span>{{ creator.Username || creator.PublicKeyBase58Check }}</span>
+            <span *ngIf="creator.IsReserved && !creator.IsVerified"
+              class="d-inline-block ml-1 cursor-pointer lh-12px fc-muted"
+              matTooltipClass="global__mat-tooltip global__mat-tooltip-font-size"
+              [matTooltip]="'This profile is reserved. Click to learn how to claim it.'">
+              <i class="far fa-clock fa-md align-middle"></i>
+            </span>
+            <span *ngIf="creator.IsVerified" (click)="tooltip.toggle()" class="ml-1 cursor-pointer text-primary"
+              matTooltipClass="global__mat-tooltip global__mat-tooltip-font-size"
+              [matTooltip]="'This account is verified'" #tooltip="matTooltip">
+              <i class="fas fa-check-circle fa-md align-middle"></i>
+            </span>
+            <span *ngIf="creator.CoinPriceBitCloutNanos">&nbsp;&middot;&nbsp;</span>
+          </div>
+          <div class="fc-muted" *ngIf="creator.CoinPriceBitCloutNanos">
+            ~{{ globalVars.nanosToUSD(creator.CoinPriceBitCloutNanos, 2) }}
+          </div>
         </div>
-        <div class="fc-muted" *ngIf="creator.CoinPriceBitCloutNanos">~{{ globalVars.nanosToUSD(creator.CoinPriceBitCloutNanos, 2) }}</div>
+      </div>
+
+      <!-- Search Cloutavista -->
+      <div class="main search-bar-cloutavista">
+        <div class="search-bar__selected-color separator-line"></div>
+        <a _ngcontent-tsu-c198="" target="_blank" class="link--unstyled fs-14px text-grey7"
```

Comment:
```suggestion
        <a target="_blank" class="link--unstyled fs-14px text-grey7"
```

Reply:
When we hover over this element, it should highlight the background like it does for the above elements in the search bar.